### ISSUE TITLE
Default reading of the card is very incomplete.

### ISSUE THREAD

**Issue by msftedad** [microsoft/vsts-extension-retrospectives#1203]
#A11y #accessibility
**Describe the bug**
Default reading of the card is very incomplete. Users with visual impairments who rely on screen readers might miss important context, actions, or information if the card content isn't fully described. 

**Are you using the Retrospectives Tool on-prem?**
No

**To Reproduce**

1. open the URL https://dev.azure.com/eDADPlayground/SCRUMProject .
2. Expand left navigation and select boards-->Retrospectives.
3. Now under the 'board' tab we have different feedback groups.
4. Press tab key to reach 'Feedback groups'
5. Verify whether screen reader reads complete information about card or not.

**Expected behavior**
Screen reader should announce the information about cards completely when focus lands on card.

**Screenshots**

https://github.com/user-attachments/assets/116324d6-3b00-4404-a34e-4f96bc049895

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/fc02f05c-aee0-469f-aa50-8c7e2c4c6cdb" />

**What screen type are you seeing this on ?**
 Desktop Windows 11

**Additional context**
